templates/chromedriver.rb

```diff
@@ -5,12 +5,18 @@
 end
 
 Capybara.register_driver :headless_chrome do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+  # Set a big (15" MBP retina resolution) window size so that
```

Comment:
suspenders is fairly [opinionated about removing configuration comments](https://github.com/thoughtbot/suspenders/blob/996fe4aa0d22931b6c3f5b01ab575ffd826c8746/lib/suspenders/app_builder.rb#L266-L292). How would you feel about removing this comment?

Reply:
Makes sense. Done in 598aa91f08874a7f9da32b25ae649b8fd1fefa2f.